api/src/main/java/io/opentelemetry/api/trace/SpanBuilder.java

```diff
@@ -44,20 +45,20 @@
  *
  * <pre>{@code
  * class MyRpcServerInterceptorListener implements RpcServerInterceptor.Listener {
- *   private static final Tracer tracer = OpenTelemetry.getTracer();
+ *   private static final Tracer tracer = OpenTelemetry.get().getTracer("com.anyco.rpc");
```

Comment:
```suggestion
 *   private static final Tracer tracer = OpenTelemetry.get().getTracer("com.example.rpc");
```
This would be the reserved domain by IANA for such purposes.